lte/gateway/release/build-magma.sh

```diff
@@ -299,13 +303,11 @@ LTE_PY_DEPS=`${RELEASE_DIR}/pydep lockfile ${RELEASE_DIR}/magma.lockfile.$OS`
 
 # now the binaries are built, we can package up everything else and build the
```

Comment:
This comment was kind of out of place before - now it is dangling. Should it be removed?